test/system/005-info.bats

```diff
@@ -305,4 +305,23 @@ EOF
     run_podman $safe_opts system reset --force
 }
 
+@test "podman - empty string defaults for certain values" {
+    skip_if_remote "Test uses nonstandard paths for c/storage directories"
+
+    # We just want this to be empty - so graph driver will be set to the empty string
+    touch $PODMAN_TMPDIR/storage.conf
+
+    # Force all custom directories so we don't pick up an existing database
+    CONTAINERS_STORAGE_CONF=$PODMAN_TMPDIR/storage.conf run_podman --root $PODMAN_TMPDIR/root --runroot $PODMAN_TMPDIR/runroot --tmpdir $PODMAN_TMPDIR/tmpdir info
```

Comment:
use `safe_opts=$(podman_isolation_opts ${PODMAN_TMPDIR})`

and then just add $safe_opts as args here and below, see the test above as example.

Second storage prints an warning for the unset driver so you need to tell the test to ignore it
```
run_podman 0+w $safe_opts info
    require_warning "The storage 'driver' option should be set" \
                    "c/storage should warn on empty storage driver"
```